frameworks/Go/fiber/benchmark_config.json

```diff
@@ -3,12 +3,15 @@
   "tests": [{
     "default": {
       "json_url": "/json",
+      "db_url": "/db",
+      "query_url": "/queries?queries=",
+      "update_url": "/update?queries=",
       "plaintext_url": "/plaintext",
       "port": 8080,
       "approach": "Realistic",
       "classification": "Platform",
-      "database": "None",
-      "framework": "Fiber",
+      "database": "MySQL",
+      "framework": "None",
```

Comment:
The rest of the PR looks good. If you could just switch this back to `Fiber` I can merge this in.

Reply:
Gotcha, thanks :+1: